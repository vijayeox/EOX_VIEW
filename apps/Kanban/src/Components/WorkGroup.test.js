import React from "react";
import { shallow, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'


import work from "./WorkGroup";
import { render } from "react-dom";
configure({ adapter: new Adapter() })
const CardInfo = [
    {
        itemId: 'item-100',
        workGroupId: "uuid1",
        itemName: 'Person-1',
        itemIcon: 'abc',
        itemDescription: 'General Manager',
        timestamp: 1622575446,
        priority: 'High',
        img:'./profile1.jpeg'
    },
    
  
]

describe('Work Group Component',()=>{
    it('renders correctly',()=>{
        const wrapper=shallow(<work/>)
        expect(wrapper).toMatchSnapshot()
          })
    it('Component is not empty',()=>{
        const wrapper=shallow(<work />)
        expect(wrapper.exists()).toBe(true)
    })
    it('Component is not empty',()=>{
        const wrapper=shallow(<work  items={CardInfo}/>)

        expect(wrapper).toHaveLength(CardInfo.length)
    })
    it('First Element',()=>{
        const wrapper=shallow(<work  items={CardInfo}/>)
        
        expect(wrapper).toHaveLength(CardInfo.length)
        console.log('*****!@#***')
        console.log(wrapper.first())
       // expect(wrapper.items.itemId).toEqual('item-10')
    })
    
})