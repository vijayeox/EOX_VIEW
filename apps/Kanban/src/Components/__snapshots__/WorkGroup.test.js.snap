// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Work Group Component renders correctly 1`] = `ShallowWrapper {}`;
